src/main/java/dk/dbc/connector/openformat/OpenFormatConnector.java

```diff
@@ -60,10 +54,12 @@ public OpenFormatConnector(Client httpClient, String baseUrl) {
      * @param baseUrl            base URL for record service endpoint
      */
     public OpenFormatConnector(FailSafeHttpClient failSafeHttpClient, String baseUrl) {
-        this.failSafeHttpClient = InvariantUtil.checkNotNullOrThrow(
-                failSafeHttpClient, "failSafeHttpClient");
-        this.baseUrl = InvariantUtil.checkNotNullNotEmptyOrThrow(
-                baseUrl, "baseUrl");
+        if (failSafeHttpClient == null || baseUrl == null) {
```

Comment:
Hmm det er jeg ikke helt sikker på hvorfor du gør? Er det fordi man ikke få linjen hvor det knækker med?

Reply:
For at slippe af med Invariant-Utils - som ikke har været rørt i 5 år :)